addons/web/static/tests/views/fields/properties_field_tests.js

```diff
@@ -561,10 +561,10 @@ QUnit.module("Fields", (hooks) => {
         await closePopover(target);
 
         await editValue("2", "2");
-        await editValue("2.11", "");
-        await editValue("azerty", "", "Wrong integer value should be interpreted as empty");
+        await editValue("2.11", "0");
+        await editValue("azerty", "0", "Wrong integer value should be interpreted as empty");
         await editValue("1,2,3,4,5,6", "123456");
-        await editValue("1,2,3,4,5,6.1,2,3", "");
+        await editValue("1,2,3,4,5,6.1,2,3", "0");
```

Comment:
Not really sure about this. Why would you fallback to 0 if the input is non-numeric? The test to be added should be:
```js
await editValue("0", "0");
await editValue("0.0", "0");
```

Reply:
I'm thinking of just keeping the current behavior. Basically, we should just focus on "zero as input".